### PR TITLE
schedule optimization

### PR DESCRIPTION
**Pull request by MegaByte875**
## What type of PR is this?
- [ ] bug
- [ ] feature
- [x] enhancement

### CODE REVIEW

**Comment by MegaByte875** [comment]
**Current:**
```
$ kubectl get cm nebula-graphd-zone -o yaml
apiVersion: v1
data:
  nebula-graphd: ""
  nebula-graphd-0: az2
  nebula-graphd-1: az3
  nebula-graphd-2: az1
  nebula-graphd-3: az1
  nebula-graphd-4: az2
  nebula-graphd-5: az2
  nebula-graphd-6: az3
  nebula-graphd-7: az1
  nebula-graphd-8: az3
  nebula-graphd-9: az3
  nebula-graphd-10: az2
  nebula-graphd-11: az1
  nebula-graphd-12: az1
  nebula-graphd-13: az2
  nebula-graphd-14: az3
  nebula-graphd-15: az2
  nebula-graphd-16: az1
  nebula-graphd-17: az3
  nebula-graphd-18: az2
  nebula-graphd-19: az3
```

**After optimization:**
```
$ kubectl get cm nebula-graphd-zone -o yaml
apiVersion: v1
data:
  nebula-graphd: ""
  nebula-graphd-0: az2
  nebula-graphd-1: az3
  nebula-graphd-2: az1
  nebula-graphd-3: az2
  nebula-graphd-4: az3
  nebula-graphd-5: az1
  nebula-graphd-6: az2
  nebula-graphd-7: az3
  nebula-graphd-8: az1
  nebula-graphd-9: az2
  nebula-graphd-10: az3
  nebula-graphd-11: az1
  nebula-graphd-12: az1
  nebula-graphd-14: az3
  nebula-graphd-15: az2
  nebula-graphd-16: az3
  nebula-graphd-17: az1
  nebula-graphd-18: az2
  nebula-graphd-19: az3
```

**Review by kqzh — APPROVED** [review]
LGTM

File: pkg/scheduler/nodezone/node_zone.go [hunk 1]
```diff
@@ -19,30 +19,42 @@ package nodezone
...
 var _ framework.EnqueueExtensions = &NodeZone{}
 
 const (
-	ConfigMap framework.GVK = "ConfigMap"
-
 	// Name is the name of the plugin used in the plugin registry and configurations.
 	Name = "NodeZone"
 
+	// AvailableZones is the number failure domains over which we should spread.
+	AvailableZones = 3
```
Comment (wenhaocs) [inline pkg/scheduler/nodezone/node_zone.go:53]: Why is it fixed?

Comment (MegaByte875) [reply pkg/scheduler/nodezone/node_zone.go:53]: I think 3 is the minimum value to ensure HA on cloud.

**Pull request merged**